man/rgbasm.5

```diff
@@ -328,76 +328,100 @@ with a non-zero constant as either operand will be constant 1, even if the other
-.It Fn MUL x y Ta Fixed-point multiplication $( x \[mu] y ) \[di] ( 2 ^ precision )$
-.It Fn FMOD x y Ta Fixed-point modulo $( x % y ) \[di] ( 2 ^ precision )$
-.It Fn POW x y Ta $x$ to the $y$ power
-.It Fn LOG x y Ta Logarithm of $x$ to the base $y$
+.It Fn DIV x y Ta Fixed-point division
+.It Fn MUL x y Ta Fixed-point multiplication
+.It Fn FMOD x y Ta Fixed-point modulo
+.It Fn POW x y Ta $x sup y$
+.It Fn LOG x y Ta $log sub y ( x )$ (such that
+.Ic LOG Ns ( Ic POW Ns ( x , y ) , y ) == x)
 .It Fn ROUND x Ta Round $x$ to the nearest integer
-.It Fn CEIL x Ta Round $x$ up to an integer
-.It Fn FLOOR x Ta Round $x$ down to an integer
+.It Fn CEIL x Ta Round $x$ up to the nearest integer
+.It Fn FLOOR x Ta Round $x$ down to the nearest integer
 .It Fn SIN x Ta Sine of $x$
 .It Fn COS x Ta Cosine of $x$
 .It Fn TAN x Ta Tangent of $x$
-.It Fn ASIN x Ta Inverse sine of $x$
-.It Fn ACOS x Ta Inverse cosine of $x$
-.It Fn ATAN x Ta Inverse tangent of $x$
+.It Fn ASIN x Ta Inverse sine of $x$ (such that
+.Ic ASIN Ns ( Ic SIN Ns ( x ) ) == x)
+.It Fn ACOS x Ta Inverse cosine of $x$ (such that
+.Ic ACOS Ns ( Ic COS Ns ( x ) ) == x)
+.It Fn ATAN x Ta Inverse tangent of $x$ (such that
+.Ic ATAN Ns ( Ic TAN Ns ( x ) ) == x)
 .It Fn ATAN2 y x Ta Angle between $( x , y )$ and $( 1 , 0 )$
 .El
 .EQ
 delim off
 .EN
 .Pp
-All of these fixed-point functions can take an optional final argument, which is the precision to use.
+Operators not in the list above, such as
+.Sq +
+and
+.Sq - ,
+work correctly with fixed-point operands as well (but
+.Em both
+operands must be fixed-point: use
+.Ql 1.0 + 2.0 ,
+not
+.Ql 1.0 + 2 ;
+and use
+.Ql MUL(1.0, 2.0)
+or
+.Ql 1.0 * 2 ,
+not
+.Ql MUL(1.0, 2) ) .
+.Pp
+All of these fixed-point functions can take an optional final argument, which is the precision to use
+.Em for the operation .
 For example,
 .Ql MUL(6.0q8, 7.0q8, 8)
 will evaluate to
 .Ql 42.0q8
 no matter what value is set as the current
 .Cm Q
 option.
+Currently,
+.Nm
+.Em does not check precisions for consistency ,
+so
+.Ql MUL(4.2q8, 6.9q12, 16)
+will produce a nonsensical result without any warnings.
 .Pp
-The trigonometry functions (
-.Ic SIN ,
-.Ic COS ,
-.Ic TAN ,
-etc) are defined in terms of a circle divided into 1.0 "turns" (equal to 2pi radians or 360 degrees).
+The trigonometry functions
+.Pq Ic SIN , Ic COS , Ic TAN , No etc
+are defined in terms of a circle divided into 1.0
+.Dq turns
+(equal to 2pi radians or 360 degrees).
 .Pp
 These functions are useful for automatic generation of various tables.
 For example:
 .Bd -literal -offset indent
-; Generate a table of sine values from sin(0.0) to sin(1.0), with
-; amplitude scaled from [-1.0, 1.0] to [0.0, 128.0]
-DEF turns = 0.0
-REPT 256
-    db MUL(64.0, SIN(turns) + 1.0) >> 16
-    DEF turns += 1.0 / 256
+; Generate a table of sine values from sin(0.0) to sin(1.0) excluded,
+; with amplitude scaled from [-1.0, 1.0] to [0.0, 128.0].
+FOR angle, 0.0, 1.0
```

Comment:
```suggestion
; Generate a table of 256 sine values from sin(0.0) to sin(1.0) excluded,
; with amplitude scaled from [-1.0, 1.0] to [0.0, 128.0].
FOR angle, 0.0, 1.0, 1.0 / 256
```

Reply:
Oops, I actually had the iteration step in my local testing copy, but forgot it when transcribing :P